### PR TITLE
added index html and styles

### PR DESCRIPTION
**Pull request by Valerie-Freeman**
Added index.html file with title and added styles.css file to style the title.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1 id="main-title">THE BEST WEBSITE EVER</h1>
+
+  <p>I'm making this us as I go!</p>
```
Comment (dalamcd) [inline index.html]: There's a typo here!

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#main-title {
+  color: goldenrod;
```
Comment (dalamcd) [inline styles.css:2]: Great color choice!

Comment (Valerie-Freeman) [reply styles.css:2]: Thanks!

File: index.html (hunk 1 above)
Comment (Valerie-Freeman) [reply index.html]: Fixed

**Review by dalamcd — APPROVED** [review]
Looks great!

**Pull request merged**